doc/stdlib/linalg.rst

```diff
@@ -101,6 +101,10 @@ Linear algebra functions in Julia are largely implemented by calling functions f
 
    Compute a factorization of a positive definite matrix ``A`` such that ``A=L*Diagonal(d)*L'`` where ``L`` is a unit lower triangular matrix and ``d`` is a vector with non-negative elements.
 
+.. function:: ldltfact(A) -> CHOLMOD.Factor
+
+   Compute the LDLt factorization of a sparse symmetric or Hermitian matrix ``A``. A fill-reducing permutation is used.  The main application of this type is to solve systems of equations with ``\``, but also the methods ``diag``, ``det``, ``logdet`` are defined. The function calls the C library CHOLMOD and many other functions from the library are wrapped but not exported.
```

Comment:
And Matlab uses `ldl` for Bunch-Kaufman, and it's especially confusing and waaay underdocumented that we're meaning a different thing with that same name. Actually I had no idea we used a totally different name for dense Bunch-Kaufman LDL^T.